src/ol_concourse/pipelines/infrastructure/keycloak/docker_packer_pulumi_pipeline.py

```diff
@@ -125,6 +125,14 @@ def build_keycloak_infrastructure_pipeline() -> PipelineFragment:
         repository="keycloak-scim",
     )
 
+    # Repo: https://github.com/keycloakify/keycloakify-starter
+    # Use: Keycloak Starter Theme
+    keycloakify_spi = github_release(
+        name=Identifier("keycloakify-spi"),
+        owner="keycloakify",
```

Comment:
I think that the intended method of using the starter theme is to fork it so that it can be used as the basis for the customized theme.